### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by Leboroz**
- Deleted all the boilerplate from CRA (text, images, styles).
- I created a directory called components.
- Inside components, I created new Calculator.js and Button.js files.
- In Calculator.js, I created a React component that matches this design:

calculator

- I used class-based components.

### CODE REVIEW

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Leboroz  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

- [x] Please separate the `App` function in your `index.js` to be a separate `class-based` `App.js` because this will be used for routing later on 😄 


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/app.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Calculator from './calculator';
+
+export default function App() {
```
Comment (rloterh) [inline src/components/app.js]: _This aspect needs further improvement:_
- [x] As mentioned in the previous review, kindly ensure that the main component _(_App.js_)_ is _class_based_.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Leo, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Please take your time to carefully improve aspects outlined in the previous code review(s), and carefully check to see that the changes are working as intended._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @Leboroz 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following Gitflow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [x] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**